docs/book/unit-testing.md

```diff
@@ -714,65 +772,74 @@ method. This test should run fine, so now we can add the rest of the test
 methods:
 
 ```php
-public function testCanDeleteAnAlbumByItsId()
-{
-    $this->tableGateway->delete(['id' => 123])->shouldBeCalled();
-    $this->albumTable->deleteAlbum(123);
-}
-
-public function testSaveAlbumWillInsertNewAlbumsIfTheyDontAlreadyHaveAnId()
-{
-    $albumData = [
-        'artist' => 'The Military Wives',
-        'title'  => 'In My Dreams'
-    ];
-    $album = new Album();
-    $album->exchangeArray($albumData);
+    public function testCanDeleteAnAlbumByItsId(): void
+    {
+        $this->tableGateway->expects($this->once())
+            ->method('delete');
+        $this->albumTable->deleteAlbum(123);
+    }
```

Comment:
The 4 spaces at the beginning are not necessary here.

Reply:
Correcte